src/context/WalletProvider/KeepKeyProvider.tsx

```diff
@@ -130,6 +151,71 @@ export const KeepKeyProvider = ({ children }: { children: React.ReactNode }): JS
     })()
   }, [keepKeyWallet, keepKeyWallet?.features, setDeviceTimeout, setHasPassphrase])
 
+  useEffect(() => {
+    if (!keepKeyWallet) return
+    ;(async () => {
+      const features = await keepKeyWallet.getFeatures()
+      const versions = await getKeepKeyVersions(keepKeyWallet, features?.bootloaderHash)
+
+      if (!versions) return
+
+      if (
+        (versions.bootloader.updateAvailable || versions.firmware.updateAvailable) &&
+        !toast.isActive(KeepKeyToastId)
+      ) {
+        toastRef.current = toast({
+          render: () => {
+            return (
+              <Alert status='info' variant='solid' colorScheme='blue'>
+                <Box alignSelf='flex-start' me={2}>
+                  <RiFlashlightLine size={24} />
+                </Box>
+                <Box>
+                  <AlertTitle>{translate('updateToast.keepKey.title')}</AlertTitle>
+                  <AlertDescription>
+                    <Text>
+                      {translate('updateToast.keepKey.newVersion')}
+                      <span> </span>
+                      <Box as='span' fontWeight='bold' color='inherit'>
+                        {translate('updateToast.keepKey.firmwareOrBootloader')}
+                      </Box>
+                      <span> </span>
+                      {translate('updateToast.keepKey.isAvailable')}
+                    </Text>
+                  </AlertDescription>
+                  <Link
+                    href={'https://beta.shapeshift.com/updater-download'}
```

Comment:
As we've now got this URL across multiple files we should make it a constant so we can easily update it.